oaknational/Oak-Web-Application
# feat: 1205 copy changed on search page, base styling and functionalit…

No search result message displays ## Description

- Added showMessage state to useFetchSearchReuslts Hook
- Extrapolated message into own NoSearchResults component

## Issue(s)

Fixes #1205 

## How to test

1. Go to https://deploy-preview-1289--oak-web-application.netlify.thenational.academy/beta/teachers/search
2. Make search for that doesn't return a result i.e. '&nspd'
3. You should see no search result message returns

## Screenshots

How it used to look (delete if n/a):
<img width="1496" alt="Screenshot 2023-02-20 at 11 58 56" src="https://user-images.githubusercontent.com/91190841/220100672-253547e4-41b9-4d52-b6cd-467e10258c00.png">


How it should now look:

<img width="1491" alt="Screenshot 2023-02-20 at 15 18 32" src="https://user-images.githubusercontent.com/91190841/220144637-75a00212-bc9b-45a0-af46-903c9b5c6f6b.png">


## Checklist

- [ x] Added or updated tests where appropriate
- [ x] Manually tested across browsers / devices
- [ x] Considered impact on accessibility
- [ x] Design sign-off
